migration.go

```diff
@@ -14,7 +14,7 @@ import (
 
 type MigrationRecord struct {
 	VersionId int64
-	TStamp    time.Time
+	TStamp    string
```

Comment:
Good catch, thank you.

Can you open a PR for README.md file and mention the solution it there?